aries_cloudagent/core/tests/test_conductor.py

```diff
@@ -323,10 +323,7 @@ async def test_outbound_message_handler_return_route(self):
             )
             assert status == OutboundSendStatus.SENT_TO_SESSION
             assert bus.events
-            assert (
-                bus.events[0][1].topic
-                == f"{EVENT_PREFIX}{OutboundSendStatus.SENT_TO_SESSION}"
-            )
+            assert bus.events[0][1].topic == status.topic
```

Comment:
I'd appreciate confirmation on this change (and the identical ones below lol), to make sure that the tests are sufficiently covering the flow. The removed code, when run with the changes to `conductor.py`, results in this error:
```
E           AssertionError: assert 'acapy::outbo..._for_delivery' == 'acapy::outbo..._FOR_DELIVERY'
E             - acapy::outbound-message::OutboundSendStatus.QUEUED_FOR_DELIVERY
E             + acapy::outbound-message::queued_for_delivery
```
The current build of the tests all pass. I'm not sure if I'm using f-strings incorrectly in the first section, or if something with the `Enum` class is weird, but I would've assumed the first option would've worked. Regardless, the second option does work, and we've already asserted that the `status` is what we expect, so I don't see any obvious problems with that, but I wanted to call attention to that.

Reply:
Oh wait, I think I understand what's happening now:
```python
assert (
                bus.events[0][1].topic
                == f"{EVENT_PREFIX}{OutboundSendStatus.SENT_TO_SESSION}"
            )
```
This would need to change to:
```python
assert (
                bus.events[0][1].topic
                == f"{EVENT_PREFIX}{OutboundSendStatus.SENT_TO_SESSION.value}"
            )
```
For the original check to work.

I think since you have the `OutboundSendStatus.SENT_TO_SESSION` check above, the new assertion you're making is good.